### PR TITLE
add(clickhouse): restore from a backup

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DDB-1125

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/clickhouse/concepts/disaster-recovery.md [hunk 1]
```diff
@@ -52,16 +55,16 @@ only changed parts are backed up and files already available in the
...
-power-on of the service. The restoration happens only for powering up a
-service after powering it down or forking a service.
-
 For more information on backups in Aiven, see
 [Backups at Aiven](/docs/platform/concepts/service_backups).
 
+### Service recovery
+
+The [restoration of a backup](/docs/products/clickhouse/howto/restore-backup) of an
+Aiven for ClickHouse service is performed on a running ClickHouse server and proceeds as
```
Comment (Khatskevich) [inline docs/products/clickhouse/concepts/disaster-recovery.md:64]: Fork is allowed even if the original clickhouse is powered off, Clickhouse server should not be running for that.

File: docs/products/clickhouse/howto/restore-backup.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+   - Backup to fork from
+   - Service name and a project
+   - Cloud provider and region
+   - Service plan and additional storage
+
+1. Click **Create fork**.
+
+Once the new fork service is running, you can change your application's connection settings
+to point to this new fork service. At this point, you can also power off the service you
```
Comment (Khatskevich) [inline docs/products/clickhouse/howto/restore-backup.md:24]: This phrase may make sense, but it confuses more, I suppose. The original service could have been powered off all the time.

**Review by Khatskevich — APPROVED** [review]
LGTM

File: docs/products/clickhouse/concepts/disaster-recovery.md [hunk 3]
```diff
@@ -52,16 +55,17 @@ only changed parts are backed up and files already available in the
...
-performed on a running ClickHouse server and proceeds as a regular
-power-on of the service. The restoration happens only for powering up a
-service after powering it down or forking a service.
-
 For more information on backups in Aiven, see
 [Backups at Aiven](/docs/platform/concepts/service_backups).
 
+### Service recovery
+
+Regardless whether your Aiven for ClickHouse service is powered on or powered off, you can
```
Comment (staceysalamon-aiven) [inline docs/products/clickhouse/concepts/disaster-recovery.md:63]:
```suggestion
Regardless of whether your Aiven for ClickHouse service is powered on or powered off, you can
```

**Review by staceysalamon-aiven — APPROVED** [review]
LGTM, only added a couple of minor suggestions

File: docs/products/clickhouse/concepts/disaster-recovery.md [hunk 4]
```diff
@@ -42,6 +42,9 @@ They cover the following:
 -   Table content (`part files`)
 -   Dictionaries
 
+You can [restore your service from a backup](/docs/products/clickhouse/howto/restore-backup)
+of your choice.
```
[the remark this answers is not in the document]
Comment (wojcik-dorota) [reply docs/products/clickhouse/concepts/disaster-recovery.md]: So far it's been only possible to restore from the latest backup so now we'd like to emphasize the users can select a backup they want.

**Pull request merged**